package.json

```diff
@@ -169,7 +169,7 @@
     "cross-env": "7.0.3",
     "cz-conventional-changelog": "3.3.0",
     "date-fns": "2.29.3",
-    "emoji-toolkit": "^7.0.0",
+    "emoji-toolkit": "7.0.0",
```

Comment:
@matthprost why is this in dept ?

Reply:
@chambo-e it's in dev deps, it is for emoji interpreter in changelog